gutenberg.php

```diff
@@ -188,6 +188,21 @@ function register_site_icon_url( $response ) {
 function gutenberg_register_widgets() {
 	if ( gutenberg_is_experiment_enabled( 'gutenberg-widget-experiments' ) ) {
 		register_widget( 'WP_Widget_Block' );
+		// By default every widget on widgets.php is wrapped with a <form>.
+		// This means that you can sometimes end up with invalid HTML, e.g. when
+		// one of the widgets is a Search block.
+		//
+		// To fix the problem, let's add a filter that moves the form below the actual
+		// widget content.
+		add_filter( 'dynamic_sidebar_params', function($arg) {
```

Comment:
Core coding standards forbid using an anonymous function in `add_action/add_filter`. Can we make this a standalone function?

Reply:
I just updated the PR to use a standalone function